hazelcast/src/main/java/com/hazelcast/spi/impl/operationexecutor/OperationExecutor.java

```diff
@@ -19,6 +19,7 @@
 import com.hazelcast.nio.Packet;
 import com.hazelcast.spi.Operation;
 import com.hazelcast.spi.impl.PartitionSpecificRunnable;
+import com.hazelcast.spi.impl.operationexecutor.impl.OperationExecutorImpl;
```

Comment:
If this import is just used in JavaDoc we will have an "unused import" error somewhere. I think it's one of the tools run by SonarQube, maybe even CheckStyle with it's separate configuration. So which side should I adjust? Should the build fail due to this or should I disable this check it in SonarQube?